variables.tf

```diff
@@ -102,7 +102,7 @@ variable "production_workspace" {
 
 variable "runtime" {
   description = "Lambda language runtime"
-  default     = "python3.11"
+  default     = "python3.12"
```

Comment:
It's possible to keep this default `null` or `""` and read the `.python-version` by default and override it if this version is supplied with something else

Reply:
```suggestion
  default     = "python3.11"
```

Reply:
Addressed https://github.com/domain-protect/terraform-aws-domain-protect/pull/77/commits/5e1f5bad94a4a4e04e68bb98346748cb4cdf1952